go.mod

```diff
@@ -1,13 +1,13 @@
 module go.starlark.net
 
-go 1.13
+go 1.17
```

Comment:
This diff doesn't seem necessary: this module doesn't require a recent version of Go.